virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
Regarding the python file name, is it better to use libvirt_disk.py ?

Reply:
Thanks, updated.